contracts/AllDay.cdc

```diff
@@ -558,6 +569,269 @@ access(all) contract AllDay: NonFungibleToken {
         AllDay.account.storage.save(setPlayTierMap, to: /storage/AllDayAdminSetPlayTierMap)
     }
 
+    //------------------------------------------------------------
+    // Badges
+    //------------------------------------------------------------  
+
+    access(all) struct Badge{
+        access(all) var slug: String
+        access(all) var title: String
+        access(all) var description: String
+        access(all) var visible: Bool
+        access(all) var slugV2: String
```

Comment:
what is the purpose of slugV2 vs slug?

Reply:
i founds it in nfl DB. I didn't get a satisfactory reason on why it exist so i decided to just add it https://dapperlabs.slack.com/archives/C081YQ1QXPH/p1755716117175149

Reply:
I see. could also still be cleaned up since we can always use the metadata map if it turns out there is any use

Reply:
since it's already implemented, i'll leave it to mirror exactly what we have on nfl